From 1a6a96789dc7d8ad2178e8a78d31e771b888900b Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Wed, 14 Jan 2015 15:55:19 +0800
Subject: [PATCH 0783/1594] MLK-10097 ARM: dts: imx6sl: add FEC sleep pinctrl
 for evk board

commit 64e0fe2cd1f033c56e6f1e7c7794289ab77a96f4 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add FEC sleep pinctrl setting for evk board.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 21f0325..709a39e 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -532,6 +532,7 @@
 		pinctrl_fec_sleep: fecgrp-sleep {
 			fsl,pins = <
 				MX6SL_PAD_FEC_MDC__GPIO4_IO23      0x3080
+				MX6SL_PAD_FEC_MDIO__GPIO4_IO20		0x3080
 				MX6SL_PAD_FEC_CRS_DV__GPIO4_IO25   0x3080
 				MX6SL_PAD_FEC_RXD0__GPIO4_IO17     0x3080
 				MX6SL_PAD_FEC_RXD1__GPIO4_IO18     0x3080
-- 
1.7.5.4

